.github/pull_request_template.md

```diff
@@ -1,23 +1,79 @@
 <!--
+A few suggestions about filling out this PR
 
-Replace this comment with a description of what's being changed by this PR.
+1. Use a descriptive title for the PR.
+2. If this pull request is work in progress, create a draft PR instead of prefixing the title with WIP.
+3. Please label this PR with one of the following labels, depending on the scope of your change:
+- New feature
+- Bug fix
+- Enhancement
+- Breaking change
+4. Remove those recommended/optional sections if you don't need them. Only "What does this PR do" and "Checklist" are mandatory.
+5. Submit the pull request: Push your local changes to your forked copy of the repository and submit a pull request (https://help.github.com/articles/using-pull-requests).
+6. Please be patient. We might not be able to review your code as fast as we would like to, but we'll do our best to dedicate it the attention it deserves. Your effort is much appreciated!
+-->
 
-If this PR should close an issue, please add one of the magic keywords
-(e.g. Fixes) followed by the issue number. For more info see:
-https://help.github.com/articles/closing-issues-using-keywords/
+## What does this PR do?
+<!-- _(Mandatory)_
+Replace this comment with a description of what's being changed by this PR. Please explain the WHAT: A clear and concise description of what (patterns used, algorithms implemented, design architecture, message processing, etc.)
+-->
+
+## Checklist
+<!-- _(Mandatory)_
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply.
```

Comment:
All users might not know that a double-tilde is a strike-through in Markdown. Perhaps we should mention it here to make it easier for users who are not familiar with Markdown?